### PR TITLE
Repair Error

### PR DESCRIPTION
**Pull request by louis1706**
should repair the bug on the master too

### CODE REVIEW

**Comment by joker-119** [comment]
What exactly is the "error" here?

**Comment by Michal78900** [comment]
Probably the CustomReasonHandler not working properly
image
I think this is going to fix an issue

File: Exiled.API/Features/Player.cs [hunk 1]
```diff
@@ -1507,7 +1507,7 @@ public void Kill(string deathReason, string cassieAnnouncement = "")
             if (Side != Side.Scp && !string.IsNullOrEmpty(cassieAnnouncement))
                 Cassie.Message(cassieAnnouncement);
 
-            ReferenceHub.playerStats.KillPlayer(new CustomReasonDamageHandler(deathReason, float.MaxValue, cassieAnnouncement));
+            ReferenceHub.playerStats.DealDamage(new CustomReasonDamageHandler(deathReason, float.MaxValue, cassieAnnouncement));
```
Comment (Michal78900) [inline Exiled.API/Features/Player.cs:1510]: Right now this method will throw NRE, you should use `new CustomReasonDamageHandler(deathReason)`

**Review by Michal78900 — CHANGES_REQUESTED** [review]
Please change the constructor I've mentioned

**Review by Michal78900 — APPROVED** [review]

**Pull request merged**